reference/src/components/popup/popup.less

```diff
@@ -6,12 +6,11 @@
     bottom: 0;
     left: 0;
     right: 0;
-    z-index: 99;
+    z-index: 1;
```

Comment:
is it tested? I think it could be that ads/other things could overlap message

Reply:
it is, apparently it's not overlaping. all good